sprocs/sync_assessments.sql

```diff
@@ -174,6 +174,17 @@ BEGIN
                 syncing_course_instance_id,
                 new_assessment_id
             ) ON CONFLICT DO NOTHING;
+            -- keep the database same as local
```

Comment:
Rather than doing two steps here (an INSERT and then an UPDATE) they should be combined in the UPSERT pattern like this:
```
INSERT INTO group_configs
    (course_instance_id, assessment_id, maximum, minimum, student_authz_create, ...etc...)
VALUES
    (syncing_course_instance_id, new_assessment_id,
    (valid_assessment.data->>'group_max_size')::integer,
    ...etc...
ON CONFLICT (assessment_id) UPDATE
SET
    maximum = EXCLUDED.maximum,
    minimum = EXCLUDED.minimum,
    ...etc...
```

For an example, see https://github.com/PrairieLearn/PrairieLearn/blob/master/middlewares/logPageView.sql#L13